src/components/communitySettings/communityPlatforms/TcCommunityPlatforms.tsx

```
@@ -1,388 +1,425 @@
+				<div className="flex flex-col space-y-3 md:flex-row md:items-center md:space-x-3 md:space-y-0">
+					<TcText text="Modules" variant="h6" fontWeight="bold" />
+					<TcText
+						text="Turn on/off modules and change their settings"
+						variant="body1"
+					/>
+				</div>
+
+				<div className="flex flex-col space-y-4 md:flex-row md:space-x-4 md:space-y-0">
+					<TcCard
+						className="max-h-[6rem] min-h-[6rem] min-w-[10rem] max-w-[10rem] flex-grow"
+						children={
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)